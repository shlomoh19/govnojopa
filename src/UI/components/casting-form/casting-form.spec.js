import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json'
import CastingForm from './casting-form'
import { act } from 'react-dom/test-utils'


describe('CastingForm', () => {
    const component = shallow(<CastingForm />)
    it('renders correctly', () => {
        const wrapper = shallow(<CastingForm />);
        expect(toJson(wrapper)).toMatchSnapshot();
    });

    it("render props Header", () => {
        const wrapper = component.find(".casting_form-wrapper");
        expect(wrapper).toHaveLength(1);
    });
});

