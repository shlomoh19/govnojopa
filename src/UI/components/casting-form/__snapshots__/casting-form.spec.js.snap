// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CastingForm renders correctly 1`] = `
<Fragment>
  <div
    className="error_wrapper"
  >
    <p
      className="input_error error_visible"
    />
    <p
      className="input_error error_visible"
    />
    <p
      className="input_error error_visible"
    />
    <p
      className="input_error error_visible"
    />
    <p
      className="input_error error_visible"
    />
    <p
      className="input_error error_visible"
    />
    <p
      className="input_error error_visible"
    />
  </div>
  <div
    className="casting_form-wrapper"
  >
    <div
      className="overlay"
      onClick={[Function]}
    />
    <div />
    <div
      className="befree-col casting__c-1"
    >
      <Input
        changeHandler={[Function]}
        error={null}
        name="name"
        placeholder="name"
        value=""
      />
      <Input
        changeHandler={[Function]}
        error={null}
        name="surname"
        placeholder="surname"
        value=""
      />
      <Input
        changeHandler={[Function]}
        error={null}
        name="email"
        placeholder="email"
        value=""
      />
      <PhoneField
        changeHandler={[Function]}
        phoneValue=""
        placeholder="phone"
      />
    </div>
    <div
      className="befree-col casting__c-2"
    >
      <LongInput
        changeHandler={[Function]}
        error={null}
        name="video1"
        placeholder="link.video"
        value=""
      />
      <LongInput
        changeHandler={[Function]}
        error={null}
        name="video2"
        placeholder="link.video"
        value=""
      />
      <div
        className="field_wrapper"
      >
        <label
          className="field_placeholder visible"
          htmlFor="input_"
        >
          transmittalLater
        </label>
        <textarea
          className="long-input-casting"
          name="about"
          onChange={[Function]}
          type="text"
        />
      </div>
    </div>
  </div>
  <div
    style={
      Object {
        "marginBottom": "14px",
        "marginTop": "20px",
        "width": "70%",
      }
    }
  >
    casting.page.end
  </div>
  <div
    className="casting_terms"
  >
    <input
      className="custom-checkbox"
      id="accept"
      type="checkbox"
    />
    <label
      htmlFor="accept"
      onClick={[Function]}
    >
       
      casting.checkbox
       
    </label>
    <p
      className="casting_terms-modal-button"
      onClick={[Function]}
    >
      Terms of agreement
    </p>
  </div>
  <Button
    disabled={true}
    onClick={[Function]}
    title="send"
  />
</Fragment>
`;
